src/components/InputForm/style.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const DivInputForm = styled.div`
+  //border: 2px solid blue;
+  margin-bottom: 10px;
+  overflow: auto;
+`;
+
+export const InputStyle = styled.input`
+  color: white;
+  background: none;
+  border: none;
+  border-right: 0px;
+  border-top: 0px;
+  boder-left: 0px;
+  boder-bottom: 0px;
+  &:focus {
+    outline: none;
+  }
+`;
+export const Rec1 = styled.div`
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  width: 0px;
+  height: 0px;
+  border-top: 10px solid aqua;
+  border-left: 10px solid aqua;
+  border-right: 10px solid aqua;
+  border-bottom: 10px solid aqua;
```

Comment:
이렇게 한 번에 작성할 수도 있답니다

```suggestion
  border: 10px solid aqua;
```

Reply:
그런데 `Rec1`는 사용하지 않는 컴포넌트인 것 같습니다...?